grammar.js

```diff
@@ -666,14 +667,16 @@ module.exports = grammar({
       '[', field('index', $._expressions), ']'
     )),
 
+    lhs_expression: $ => choice(
```

Comment:
Could we change this to `_lhs_expression` to hide this node in the output?

Reply:
Sure